UI_test_task_1/spec/spec_helper.rb

```diff
@@ -8,7 +8,22 @@
 require_relative '../pages/main_page'
 require_relative '../drivers/driver'
 
+# Bugs
```

Comment:
I don't get it. As I'm understood, we should create a class which consist an array of the main page methods, but the problem connects with that fact that main_pg is not understandable object. 

Do we need to add a class of the main page or we should use Inheritance to the StartUI class?

Reply:
add the method to spec file and use it in before, when you already have access to the page itself